src/libos/src/poll/do_epoll/epoll_file.rs

```diff
@@ -167,7 +167,11 @@ impl EpollFile {
             // If no ready entries for now, wait for them
             if poller.is_none() {
                 poller = Some(Poller::new());
-                self.pollee.poll(Events::IN, poller.as_mut());
+            }
+
+            let events = self.pollee.poll(Events::IN, poller.as_mut());
+            if !events.is_empty() {
+                continue;
```

Comment:
I didn't get it. I think the changed code is equivalent to the original version.